test/gtest/Makefile.am

```diff
@@ -70,6 +70,7 @@ gtest_SOURCES = \
 	common/gtest-all.cc \
 	common/main.cc \
 	common/test_gtest_cmn.cc \
+	common/mem_buffer.cc \
```

Comment:
strange commit... shouldn't it be in another PR?

Reply:
This looks like a bug in GitHub (?)

Here it shows:
![image](https://user-images.githubusercontent.com/11650339/62157752-91e70680-b316-11e9-8f9d-691547556520.png)

But the real change is:
![image](https://user-images.githubusercontent.com/11650339/62157812-b2af5c00-b316-11e9-8dad-547a84728fff.png)